newdle/client/src/components/home/HomePage.js

```diff
@@ -17,20 +18,26 @@ export default function HomePage() {
   return (
     <div>
       <div className={styles.container}>
-        <h2>How does it work?</h2>
+        <h2>
+          <Trans>How does it work?</Trans>
+        </h2>
         <ol className={styles.box}>
           <li>
             <img src={participantsIcon} alt="" className={styles.icon} />
-            <span>Choose your participants</span>
+            <span>
+              <Trans>Choose your participants</Trans>
+            </span>
           </li>
           <li>
             <img src={calendarIcon} alt="" className={styles.icon} />
-            <span>Set the time slots based on their availability</span>
+            <span>
+              <Trans>Set the time slots based on their availability</Trans>
+            </span>
           </li>
           <li>
             <img src={answersIcon} alt="" className={styles.icon} />
             <span>
-              <strong>newdle</strong> will collect the answers!
+              <strong>newdle</strong> <Trans>will collect the answers!</Trans>
```

Comment:
Can we somehow include newdle as a placeholder in the string? While it should never be translated, concatenating a translatable string won't work in any language where newdle isn't the first word in this sentence?

If it's not possible let's just include the `<strong>newdle</strong>` in the trans and hope translators don't try translating it ;)

Reply:
If you are going to translate it with an automatic translator, I can create a variable named with numbers like _0, and use it in the string. 
In that way, the languages that not use 'newdle' as the first word could be correctly translated.
If I write the code like this: 
```js
const _0 = 'newdle';
<Trans><strong>{_0}</strong> is a collective meeting scheduling application.</Trans>
```
it will generate this:
`"<0>{_0}</0> is a collective meeting scheduling application."`
So you can move the variable inside the numbered tag wherever you want, for example, you could translate it like this:
`"A collective meeting scheduling application: <0>{_0}</0>"`
and it will generate:
`A collective meeting scheduling application: <strong>newdle</strong>`
Is this what you mean? :)

Reply:
> If you are going to translate it with an automatic translator

we'll never do that :) I was more thinking about someone accidentally thinking it's a word that should be translated

I guess `<Trans><strong>{'newdle'}</strong> is a collective meeting scheduling application.</Trans>` won't work? In that case I'd just keep `newdle` as-is and assume translators realize that the product's name should not be translated